package.json

```diff
@@ -61,5 +61,9 @@
     "ts-jest": "^27.1.3",
     "tslib": "^2.3.1",
     "typescript": "^4.5.5"
+  },
+  "lint-staged": {
```

Comment:
suggestion (non-blocking): perhaps we could move it in a separate file [like this](https://github.com/illright/draw-ill-help/blob/main/.lintstagedrc.yaml)? In that example, there are also slightly better commands, you may wanna use them instead.